tools/regenerate-snapshots.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Copyright 2022-2023 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+
+## Helper script to regenerate insta snapshots from scratch.
+
+set -eu
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+cd "$SCRIPT_DIR/.."
+
+
+if [ $(git status --porcelain | wc -l) -ne 0 ]; then
```

Comment:
```suggestion
if [ "$(git status --porcelain | wc -l)" -ne 0 ]; then
```
(this is a shellcheck warning, someday that will be part of CI).

Reply:
sounds good. it's a false positive since `wc -l` returns no spaces, but I'm happy to fix it.